examples/README.md

```diff
@@ -0,0 +1,17 @@
+# README for FreeRTOS-Kernel/examples
+
+The easiest way to use FreeRTOS is to start with one of the pre-configured demo application projects.
+See [FreeRTOS/FreeRTOS/Demo](https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS/Demo) to find a list of pre-configured demos on multiple platforms which demonstrate the working of the FreeRTOS-Kernel.
+This directory aims to further facilitate the beginners in building their first FreeRTOS project.
+
+
+## Directory Structure:
+
+* The [cmake_example](./cmake_example) directory contains a minimal freertos example project , which uses the configuration file in the template_configuration directory listed below. This will provide you a starting point for building you applications using FreeRTOS-Kernel.
+* The [coverity](./coverity) directory contains a project to run Synopsys Coverity for checking MISRA compliance. This directory contains further readme files and links to documentation.
```

Comment:
Can we add Synopsys Coverity link here ? https://www.synopsys.com/software-integrity/static-analysis-tools-sast/coverity.html

Reply:
Done